src/components/Timeline/EventFilter.js

```diff
@@ -31,11 +31,15 @@ export const filterSearchForm = {
 }
 
 
-const EventFilter = ({pagination, filter, filterSearchField, eventTypes, filterTypes, dispatch, history}) =>  {
-  const filterChips = filter && filter.eventTypes && eventTypes ? renderChips(history, filter, eventTypes, dispatch): null
+const EventFilter = ({pagination, filter, filterSearchField, filterTypes, dispatch, history}) =>  {
   return  (
     <div className="incident-EventFilter">
-      {filterChips}
+      <FilterChips
+        selectSpecificFilter={'eventTypes'}
+        lookupFilterObject={'events.filter'}
+        recordLookup={'eventTypes.records'}
+        onRequestDelete={(filter, id) => () => dispatch(filterActions.removeFilter(history, 'eventTypes')(filter,id))}
+      />
       <AutoComplete
         floatingLabelText="Filter by event type"
         filter={AutoComplete.caseInsensitiveFilter}
```

Comment:
There's still a bug in onNewRequest, because the object passed into the function can be one of two things: when someone clicks on the list, it gets the eventType object selected (which works), but when someone presses enter, it gets just the text (example: 'Note'), and dispatch fails.
  
I suggest pulling out the onNewRequest function into a defined const with a signature like 

const onNewRequest = (filterTypes) => (inputResult) => {
Which makes the argument to AutoComplete

onNewRequest={onNewRequest(filterTypes)}

Then in onNewRequest you can check whether inputResult is an object or a string, and if it's a string you can use the string to lookup the eventType by name in filterTypes.

Reply:
Actually, probably better to fix this one in https://github.com/Azure/Sia-EventUI/pull/60 , so no changes needed for that on this pull request